t/page/all_countries.rb

```diff
@@ -0,0 +1,20 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/all_countries'
+
+describe 'AllCountries' do
+  subject do
+    Page::AllCountries.new
+  end
+
+  it 'should return an array' do
+    subject.world.must_be_instance_of Array
+  end
+
+  it 'should return a list of countries' do
+    countries = subject.world.flatten
```

Comment:
Why do you need to `flatten` this? If the actual structure coming back here is complex enough to need that, we might need a test for the real structure.